### PR TITLE
Spellcheck node_operation.asciidoc

### PR DESCRIPTION
**Pull request by HenryWait**
Several spelling and typo fixes.

### CODE REVIEW

File: node_operations.asciidoc [hunk 1]
```diff
@@ -518,7 +518,7 @@ To secure your operating system, here are some of the top items to consider:
 . Maintenance - Make sure that you keep your operating system up to date. Enable automated daily or weekly installation of security updates.
 . Least Privilege - Set up users for specific processes and give them the least access needed to run a service. Do not run processes with admin privileges (e.g. root).
 . Process Isolation - Use the operating system features to isolate processes from each other.
-. File System Permissions - Configure the file system carefully, on the least-privilege principle. Do not make files readable or writeable by everyone.
+. File System Permissions - Configure the file system carefully, on the least-privilege principle. Do not make files readable or writable by everyone.
```
Comment (keblek) [inline node_operations.asciidoc:521]: is this correct?

**Review by keblek — CHANGES_REQUESTED** [review]
looks good except writeable?

**Comment by HenryWait** [comment]
> looks good except writeable?

Writeable is sometimes an alternative way to write writable however most dictionaries and written expamples of the word probably have writable, writable was the way I was taught to spell it in school. 

https://wikidiff.com/writeable/writable

File: node_operations.asciidoc (hunk 1 above)
Comment (HenryWait) [reply node_operations.asciidoc:521]: See my other comment, 
Also search the chmod man page for "writable" you can find it there. 
https://www.man7.org/linux/man-pages/man1/chmod.1.html

**Comment by keblek** [comment]
Minor difference anyway. Thanks!

**Review by renepickhardt — APPROVED** [review]
LGTM

**Pull request merged**